recipes/theseus/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "theseus" %}
+{% set version = "3.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://theobald.brandeis.edu/theseus/src/{{ name }}_{{ version }}.tar.gz
+  sha256: 3cfd4f906717f9cb8e77f689fc97059b32df355c0696077034ea99a485e3f2fe
+
+build:
+  number: 0
```

Comment:
If you're only supplying a `build.sh` and therefore implying that a Windows build is not supported, please add:

```
skip: True  # [win]
```

under `number` in the `build` section.

Reply:
a windows build would require a `bld.bat` file equivalent to the `build.sh`